js-intermediate.md

```diff
@@ -5,102 +5,94 @@ Answer the following questions. First, without external resources. Challenge you
-  
-  A class can be used several times, while an ID can only be used once, so you should use classes for items that you know you're going to use a lot.
+  Researched answer:
 
 
 4. Name 4 semantic HTML tags.
 
-  Your answer:not sure what these are.
+  Your answer:
 
-  Researched answer: <header>, <p>, <footer>, <section>, <article>
+  Researched answer:
 
 
 5. What are three options for creating responsive design?
 
-  Your answer: Using a template,resourcing out to a website formats. 
+  Your answer:
 
   Researched answer:
-  
-  Flexible layouts - Using a flexible grid to create the website layout that will dynamically resize to any width.
-  
-  Media queries - An extension to media types when targeting and including styles. Media queries allow designers to specify different styles for specific browser and device circumstances.
-  
-  Flexible media - Makes media (images, video and other formats) scalable, by changing the size of the media as the size of the viewport changes.
 
 
 ### Stretch: The following questions are potential interview questions. First, try to answer each question on your own then Google the answer to further your knowledge.
 
 1. What is front end development? Can you identify any tools/skills that are uniquely required of front end developers?
 
-  Your answer:front end development has to do with the user interface part of coding. That could have to do with what the user see's when turning on their Monitor.
+  Your answer:
 
-  Researched answer: Front-end web development, also known as client-side development is the practice of producing HTML, CSS and JavaScript for a website or Web Application so that a user can see and interact with them directly.
+  Researched answer:
 
 
 2. What is block scope in JavaScript?
 
   Your answer:
 
-  Researched answer: A block scope is the area within if, switch conditions or for and while loops. Generally speaking, whenever you see {curly brackets}, it is a block. In ES6, const and let keywords allow developers to declare variables in the block scope, which means those variables exist only within the corresponding block.
+  Researched answer:
 
 
 3. How would you explain the idea of "inheritance" in object oriented programming?
 
-  Your answer: When working in React or JavaScript you have child/dumb components that will be passed down or will inherit traits from the Parent/Smart component (like a family tree).
+  Your answer:
 
-  Researched answer:Inheritance is a mechanism in which one class acquires the property of another class. For example, a child inherits the traits of his/her parents. With inheritance, we can reuse the fields and methods of the existing class. Hence, inheritance facilitates Reusability and is an important concept of OOPs.
+  Researched answer:
```

Comment:
These researched answers are fantastic. I don't have much more to add. You hit the key words and concepts I was looking for. 
- Client-side development
- Variables existing only within the scope of their block
- Reusing methods with inheritance 
Great job!